sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingAggregationSuite.scala

```diff
@@ -53,7 +54,30 @@ class StreamingAggregationSuite extends StateStoreMetricsTest
 
   import testImplicits._
 
-  test("simple count, update mode") {
+  val confAndTestNamePostfixMatrix = List(
```

Comment:
Can we do this within beforeAll and afterAll with spark.conf.set / spark.conf.unset?

Reply:
`withSQLConf` looks like used widely between SQL unit tests, and does additional work (SparkSession.setActiveSession), so I'm not sure it will work technically same. Moreover, we need to run same test "multiple times", with changing configuration.

Could you propose your code if you don't really mind? Thanks in advance!

Reply:
Oops. "we need to run same test "multiple times", with changing configuration." I missed this.
We could consider like:

https://github.com/apache/spark/blob/489a5294d106130beda1509e3cbbaf707a3d703d/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala#L61-L72

e.g.,

```
StreamingAggregationSuite {
  override def afterAll(): Unit = {
    // false
  }

  override def beforeAll(): Unit = {
    // false
  }
}

RemoveRedundantStreamingAggregationSuite extends StreamingAggregationSuite {
  override def afterAll(): Unit = {
    // true
  }

  override def beforeAll(): Unit = {
    // true
  }
}
```

but I believe the current implementation works too in this case.

Reply:
OK. I'd like to wait for other reviewers regarding opinions/suggestions on this. Let me keep this as it is until then.